akka-http-core/src/main/resources/reference.conf

```diff
@@ -18,6 +18,15 @@ akka.http {
     # the request, no `Server` header will be rendered at all.
     server-header = akka-http/${akka.http.version}
 
+    # EXPERIMENTAL FEATURE: Bind using HTTP/2
```

Comment:
Yes, it's not quite ready for production prime time, so let's use "preview" I think. Also not to revive the old experimental wording anymore - will change